docs/stylesheets/extra.css

```diff
@@ -1,3 +1,4 @@
-.md-header {
-    background-color: rgb(203, 137, 70);
-}
\ No newline at end of file
+:root {
+    --md-primary-fg-color: rgb(219, 149, 80);
+    --md-accent-fg-color: rgb(144, 180, 254);
+  }
```

Comment:
That's a funny indentation of the closing bracket. 😄 I'll merge as it is, but maybe we should ship the repo with some `.vscode` settings that makes it easier to keep some standard indentation...

Reply:
This is pretty standard in CSS world though.